### ISSUE TITLE
<script setup lang="ts" generic=" multiline generic definition "> cause ESLint Parsing error, but can correctly build.

### ISSUE THREAD

**Issue by pronitsateli731** [vuejs/eslint-config-typescript#66]
This code cause error  `Parsing error: ';' expected.` at TBar comment row.
It is not occur not only when comment row exists, but also when generic definition has line breaks.

``` vue
<script setup lang="ts" generic="
    // comment of TFoo
    TFoo extends Record<string, any>,
    // comment of TBar
    TBar
">
  ...
</script>
```

I write a component which has 6 generics and each definition lasting about 50 characters, so generic definition has 13 lines.
Vue.js can parse this and can built and move correctly, but only ESLint displays error.

ESLint seems not responsible for Vue, so I submit this bug? to here.
If more suitable place exsits, I'm please to tell me that and sorry for your bother.

**Comment by pronitsateli731** [comment]
I noticed this issue is a problem on parser, not this config, so I close this.
See https://github.com/vuejs/vue-eslint-parser/issues/21
